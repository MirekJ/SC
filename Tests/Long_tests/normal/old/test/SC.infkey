53afc15d-54b9-914e-1ef8-4ed7652bc6ed
