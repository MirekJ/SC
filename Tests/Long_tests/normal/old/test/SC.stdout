
Patchy Spherocylinders version 3.5 
-------------------------------------
Reading options...
 Pressure coupling type:                             1
 Pressure:                                           0.00000000
 Replica exchange pressure:                          0.00000000
 Average volume change attempts per sweep:           0.00000000
 Equilibration sweeps:                               0
 Sweeps between step size adjustments:               0
 Production sweeps:                                  30000000
 Sweeps between statistics samples:                  50000
 Sweeps between statistics reports:                  50000
 Average chain move attempts per sweep:              0.00000000
 Initial maximum displacement:                       0.21200000
 Inititial maximum angular change (degrees):         7.50000000
 Inititial maximum box edge change:                  0.00000000
 Initial maximum chain displacement:                 0.00000000
 Inititial maximum chain angular change (degrees):   0.00000000
 Temperature in kT/e:                                0.24426000
 Parallel tempering temperature in kT/e:             0.28426000
 Sweeps between replica exchange:                    0
 Wang-Landau method:                                 0 0
 Calculate the Wang-Landau method for atom type:     0
 Average type switch attempts per sweep:             0.00000000
 Number of Sweeps per pairlist update:               0
 Random number seed:                                 196150684
 Number of sweeps per writing out cluster info:      50000
 Sweeps between movie frames:                      50000

Initialize chainlist...
Reading topology...
Topology read of 1: A (geotype: CPSC, 13) with parameters 1.333333 1.200000 1.346954 0.300000 60.000000 5.000000 
Topology read for molecule: A 
particle 1: 	1 
system: A 164

Topology succesfully read. Generating pair interactions...
Generating chainlist...
Generating connectivity...

Reading configuration...
Equilibration of maximum step sizes:  0 sweeps

Initializing energy functions...
Production run:  30000000 sweeps

Energy drift: -0.000121513561680 
Starting energy+pv: -6336.60873881 
Starting energy: -302.46365330 
Deallocating memory...

Done

